packages/augur-test/tests/state/api/Markets.test.ts

```diff
@@ -0,0 +1,193 @@
+import { API } from "@augurproject/state/src/api/API";
+import { MarketInfo } from "@augurproject/state/src/api/Markets";
+import { MarketType } from "@augurproject/state/src/api/types";
+import { DB } from "@augurproject/state/src/db/DB";
+import {
+  ACCOUNTS,
+  makeDbMock,
+  compileAndDeployToGanache,
+  ContractAPI,
+} from "../../../libs";
+import { stringTo32ByteHex } from "../../../libs/Utils";
+
+const mock = makeDbMock();
+
+beforeEach(async () => {
+  mock.cancelFail();
+});
+
+let db: DB<any>;
+let api: API<any>;
+let john: ContractAPI;
+let mary: ContractAPI;
+
+beforeAll(async () => {
+  const {provider, addresses} = await compileAndDeployToGanache(ACCOUNTS);
+
+  john = await ContractAPI.userWrapper(ACCOUNTS, 0, provider, addresses);
+  mary = await ContractAPI.userWrapper(ACCOUNTS, 0, provider, addresses);
```

Comment:
Since you're keeping mary, make sure to use ACCOUNTS index `1` since john is already using `0`.

```suggestion
  mary = await ContractAPI.userWrapper(ACCOUNTS, 1, provider, addresses);
```

Reply:
Oops, thanks. I copied from the line above and then forgot to update the second param.